### DISCUSSION TITLE
[Multiplay] New permission - open/close all ride

### DISCUSSION THREAD

**Discussion by Lastorder-DC** [OpenRCT2/OpenRCT2#15697] [Feature requests]
Players can modify rides also can open/close all ride/shop. this can abused easily.
batch placing objects are seperate permission, so open/close all ride/shops should be seperate permission.
- [ ] Reproducible in RCT2 (vanilla)?
- [x] Multiplayer?

**Comment by Nubbie** [comment]
This should be easy to fix
